src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -2223,23 +2224,36 @@ public static function sendContactUndo($target, $cid, $uid)
 		$id = DI::baseUrl() . '/activity/' . System::createGUID();
 
 		$owner = User::getOwnerDataById($uid);
-		$data = ['@context' => ActivityPub::CONTEXT,
+		$data = [
+			'@context' => ActivityPub::CONTEXT,
 			'id' => $id,
 			'type' => 'Undo',
 			'actor' => $owner['url'],
-			'object' => ['id' => $object_id, 'type' => 'Follow',
+			'object' => [
+				'id' => $object_id,
+				'type' => 'Follow',
 				'actor' => $owner['url'],
-				'object' => $profile['url']],
+				'object' => $profile['url']
+			],
 			'instrument' => self::getService(),
-			'to' => [$profile['url']]];
+			'to' => [$profile['url']]
```

Comment:
Adding a trailing comma after each array element, even the last one, prevents possible syntax errors when adding more elements and limit the number of changed lines in git diffs.
```suggestion
			'to' => [$profile['url']],
```

Reply:
Should be fixed by now.